vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**suggestion**: Consider adding more comments or documentation links to explain the configuration options used. This can help new developers understand the setup more easily.